### DISCUSSION TITLE
Shift Click

### DISCUSSION THREAD

**Discussion by FTYears** [PrismarineJS/mineflayer#2560] [Q&A]
I view alomst all the passage about shift click but I donot find out if it can work now. Can  anyone tell me and teach me how to use it if it is useable now? Thanks a lot.

**Comment by amoraschi** [comment]
By shift click do you mean placing a block while sneaking or shift clicking in a window?

**Reply by FTYears** [reply]
I want to click a specific item  of a menu in a server, it should use shift + right click

**Reply by amoraschi** [reply]
Shift-clicking isn't implemented, but you can still click normally with bot.clickWindow

**Reply by FTYears** [reply]
Emmm that means I have no method to shift click the menu, right?

**Reply by amoraschi** [reply]
You can still send the packet with
```js
bot._client.write('packet_name', <data>)
```

**Reply by FTYears** [reply]
Sorry for disturbing you, but I want to know how to use bot._client to do that.Can you teach me or give me a tutorial web?

**Reply by amoraschi** [reply]
Look at the unstable api docs

You can also use Pakkit to look at which packets are being sent

**Reply by FTYears** [reply]
I have no idea how to do it. Wish for someone to help :(
